doc/releases/release-notes-3.6.rst

```diff
@@ -308,14 +262,13 @@ Build system and infrastructure
 
 * Fixed an issue where the macros ``_POSIX_C_SOURCE`` and ``_XOPEN_SOURCE`` would be defined
   globally when building with Picolibc or for the native (``ARCH_POSIX``) targets.
-  After this change users may need to define them for their own applications or libraries if they
-  require them.
+  After this change users may need to define them for their own applications or libraries.
 
-* Added support for sysbuild setting a signing script (``SIGNING_SCRIPT``), see
+* Added support for sysbuild setting a signing script (``SIGNING_SCRIPT``). See
   :ref:`west-extending-signing` for details.
 
 * Added support for ``FILE_SUFFIX`` in the build system which allows for adding suffixes to
-  application Kconfig fragment file names and devicetree overlay file names, see
+  application Kconfig fragment file names and devicetree overlay file names. See
   :ref:`application-file-suffixes` and :ref:`sysbuild_file_suffixes` for details.
 
 * Deprecated ``CONF_FILE`` ``prj_<build>.conf`` build type.
```

Comment:
line 287: 
"Enabling all instances in same application" -> "Enabling all instances in +the+ same application"